### ISSUE TITLE
Add MMseqs2 as align protocol

### ISSUE THREAD

**Issue by sacdallago** [debbiemarkslab/EVcouplings#218]
@thomashopf don't know how you feel about this, but I think this could be an interesting development for the future in case EVcouplings can be run on a bigger machine

**Comment by thomashopf** [comment]
Yes would be great to bring the Söding lab tools back in with additional protocols
